### ISSUE TITLE
Wrong year entered for 05/01/2023

### ISSUE THREAD

**Issue by fenjas** [COVID19-Malta/COVID19-Data#23]
05/01/2022 instead of 05/01/2023

![image](https://user-images.githubusercontent.com/19566745/210990323-ba9d942b-b1c3-49c0-8875-79bf97f0e75d.png)

**Comment by Kassandra8** [comment]
Hi, do you know where we can find today's data as it seems to be missing from the page?

**Comment by fenjas** [comment]
No, sorry. I read data directly from the csv they usyally update daily.
